ape/logic.py

```diff
@@ -83,16 +86,18 @@ def execute_mgmt_api_request(method, url, data_object=None):
         'Authorization': f'Bearer {mgmt_token}',
         'Content-Type': 'application/json'
     }
-    auth0_domain = util.env.get("AUTH0_DOMAIN")
+    auth0_domain = env.get("AUTH0_DOMAIN")
     data = json.dumps(data_object) if data_object else None
-    result = requests.request(method=method, url=f'https://{auth0_domain}{url}', headers=headers, data=data)
+    url = f'https://{auth0_domain}{url}'
```

Comment:
You're overwriting `url` variable. Moreover, it's used as a param in `execute_mgmt_api_request` earlier and then shadowed here with a new value. Could this be simplified a little bit? It is a little hard to quickly understand why you're making two separate calls to the users api endpoint.

Reply:
Fixed.